### DISCUSSION TITLE
Visual Studio 2022 Shortcut-Tooltip covers suggestion

### DISCUSSION THREAD

**Discussion by DanzigGLT** [community/community#18109] [Copilot Conversations] [answered]
The tooltip with the shortcuts (Tabstop to accept | Alt+. next) is covering part of the
suggestion (most of the time directly where the caret is but not always) and should be 
placed just "1 line above" to be happy. Got that problem on two different installations 
(VS2022 CE / Prof). It also pops up when there is no suggestion at all (in that case
always at caret-position). I'm puzzled, I know several programmers who use it without problems,
only "difference" I can spot is that I'm the only one who has also the C++ Workload installed.

![image](https://user-images.githubusercontent.com/58229407/172699911-114f4672-00bd-4555-a88b-af02587f7ac3.png)

Here is an example of empty suggestion just covering (hiding actually) what I'm typing.

![image](https://user-images.githubusercontent.com/58229407/172702743-5d4f15f0-8686-477f-b048-d179293d1c74.png)

**Comment by daniellovell** [comment, marked as answer]
This is a duplicate of #17864. I am having a similar issue though, and also have the C++ workload installed.

**Reply by DanzigGLT** [reply]
I was searching for this but didn't find it :-( Thanks for pointing me to the right direction.
